src/components/Choropleth.vue

```diff
@@ -16,80 +16,79 @@
                 <ResetMapButton :selected-indicator="mapSettings.indicator" @reset-view="updateRegionBounds" />
             </LControl>
             <LControl position="topright">
-                <AdminLevelToggle @change-admin-level="handleChangeAdminLevel" v-if="mapSettings.country" />
                 <HelpAlert />
             </LControl>
             <LControl position="topright">
                 <ExcelDownloadButton />
             </LControl>
+            <LControl v-if="isLargeScreen" position="bottomleft" style="margin-left: 2rem; margin-bottom: 2rem">
+                <map-settings-menu :is-large-screen="true"></map-settings-menu>
+            </LControl>
         </LMap>
+        <div v-if="!isLargeScreen">
+            <map-settings-menu :is-large-screen="false"></map-settings-menu>
+        </div>
```

Comment:
On mobile, the gap looks weird because of the map's attribution:
![image](https://github.com/mrc-ide/arbomap/assets/60350599/9355ecd0-5880-48b3-8ccb-a7ae33ffd183)